tests/helpers.py

```diff
@@ -395,153 +394,103 @@ def validate_cephfilesystem(fs_name):
     )
     ct_pod = pod.get_ceph_tools_pod()
     ceph_validate = False
-    k8s_validate = False
+    ocp_validate = False
     cmd = "ceph fs ls"
-    logger.info(fs_name)
+
+    result = CFS.get(resource_name=fs_name)
+    if result.get('metadata').get('name'):
```

Comment:
It would be better to use TimeoutSampler instead of retry, as done in tests.helpers.verify_block_pool_exists

Reply:
Done